src/common/models/model.h

```diff
@@ -67,11 +69,12 @@ class FModel
 	virtual ~FModel();
 
 	virtual bool Load(const char * fn, int lumpnum, const char * buffer, int length) = 0;
-	virtual int FindFrame(const char * name, bool nodefault = false) = 0;
```

Comment:
Please avoid any change to formatting. Such useless noise only complicates diffing and bisecting. This is especially important for huge PR like this one.